### PR TITLE
unittest added for utf8, sshkey and hostkey

### PR DESCRIPTION
**Pull request by arif-pragmasys**

### CODE REVIEW

File: regress/unittests/utf8/tests.c [hunk 1]
```diff
@@ -9,7 +9,7 @@
 #include <locale.h>
 #include <string.h>
 
-#include "test_helper.h"
+#include "../test_helper/test_helper.h"
```
Comment (manojampalam) [inline regress/unittests/utf8/tests.c:12]: This looks like Windows specific change. The intent is to try to keep the source same wherever possible or add WINDOWS #def around. In this case, looks like adding test_helper directory to INCLUDE in project file should do the trick?

File: regress/unittests/win32compat/test_helper.c [hunk 2]
```diff
@@ -43,6 +43,8 @@
 #include "test_helper.h"
 //#include "atomicio.h"
 
+#define F_OK 0
```
Comment (manojampalam) [inline regress/unittests/win32compat/test_helper.c:46]: IS this a Windows specific change. If so, add #ifdef WINDOWS around

**Review by manojampalam — APPROVED** [review]
Take care of these comments on your next Pull Request. I'll be merging this Pull Request now.

File: regress/unittests/win32compat/test_helper.c (hunk 2 above)
Comment (arif-pragmasys) [reply regress/unittests/win32compat/test_helper.c:46]: this file is in the win32compat directory though, do we still need #ifdef Windows here?

File: regress/unittests/utf8/tests.c (hunk 1 above)
Comment (arif-pragmasys) [reply regress/unittests/utf8/tests.c:12]: The file in 'openssh/open-ssh portable' repo has it this way. I believe during our first porting we modified it (got rid of ../test_helper/ part)

**Pull request merged**